tests/test.js

```diff
@@ -6,7 +6,10 @@ function format(str, options = {}) {
   return prettier
     .format(str, {
       pluginSearchDirs: [__dirname], // disable plugin autoload
-      plugins: [path.resolve(__dirname, '..')],
+      plugins: [
+        path.resolve(__dirname, '..'),
+        require('prettier-plugin-astro'),
```

Comment:
This need to be on top of the `path.resolve` and it needs to be `require.resolve` instead of just `require`

Reply:
🤦  thanks